e2etest/zt_managed_disks_test.go

```diff
@@ -147,3 +147,61 @@ func TestManagedDisks_OAuthRequired(t *testing.T) {
 		"",
 	)
 }
+
+// Test a managed disk of size 7.9 TB
+func TestManagedDisks_LargeManagedDisk(t *testing.T) {
```

Comment:
Are we trying to download an 8TB md in the pipeline? will that even work? this will probably also eat up waaaay too much time if we run this live.

Reply:
That's correct and that is a valid concern. I could potentially skip the test entirely and/or pursue testing in unit tests or new e2e.